dashboard/test/ui/features/learning_platform/teacher_homepage.feature

```diff
@@ -14,16 +14,6 @@ Feature: Using the teacher homepage sections feature
     When I create a new section and go home
     Then the section table should have 2 rows
 
-  Scenario: Loading teacher homepage with course experiment enabled
```

Comment:
Based on conversation with Dave and Tess, we are removing the ability to assign alternative units.